pages/_app.js

```diff
@@ -1,18 +1,24 @@
 import GlobalStyle from "../styles";
 import { initialOutivities } from "@/lib/data";
 import Layout from "@/components/Layout";
-
 import useLocalStorageState from "use-local-storage-state";
+import { useRouter } from "next/router";
 
 export default function App({ Component, pageProps }) {
   const [outivities, setOutivities] = useLocalStorageState("outivities", {
     defaultValue: initialOutivities,
   });
 
+  const router = useRouter();
```

Comment:
Do you need useRouter() here?

Reply:
Yeah true it's not needed, thank you for helping to clean up 🧹